sky/execution.py

```diff
@@ -699,14 +699,22 @@ def _maybe_translate_local_file_mounts_and_sync_up(
     original_storage_mounts = task.storage_mounts if task.storage_mounts else {}
 
     copy_mounts = task.get_local_to_remote_file_mounts()
+    has_local_source_paths_file_mounts = True
+    has_local_source_paths_workdir = task.workdir is not None
     if copy_mounts is None:
         copy_mounts = {}
-
-    has_local_source_paths = (task.workdir is not None) or copy_mounts
-    if has_local_source_paths:
-        logger.info(
-            f'{colorama.Fore.YELLOW}Translating file_mounts with local '
-            f'source paths to SkyPilot Storage...{colorama.Style.RESET_ALL}')
+        has_local_source_paths_file_mounts = False
```

Comment:
Wondering if it would be cleaner to just do `has_local_source_paths_file_mounts = bool(copy_mounts)` after the above `if` block